pixi.toml

```diff
@@ -33,12 +31,19 @@ scipy = "*"
 shapely = "*"
 xmipy = "*"
 
-[pypi-dependencies]
-flopy = { git = "https://github.com/modflowpy/flopy.git" }
-modflowapi = { git = "https://github.com/MODFLOW-USGS/modflowapi.git" }
-modflow-devtools = { git = "https://github.com/MODFLOW-USGS/modflow-devtools.git" }
-
 [tasks]
+# install
+install-flopy = "pip install --no-build-isolation --no-deps --disable-pip-version-check git+https://github.com/modflowpy/flopy.git"
+install-pymake = "pip install --no-build-isolation --no-deps --disable-pip-version-check git+https://github.com/modflowpy/pymake.git"
+install-modflowapi = "pip install --no-build-isolation --no-deps --disable-pip-version-check git+https://github.com/MODFLOW-USGS/modflowapi.git"
+install-modflow-devtools = "pip install --no-build-isolation --no-deps --disable-pip-version-check git+https://github.com/MODFLOW-USGS/modflow-devtools.git"
+install = { depends_on = [
```

Comment:
I don't mind `install` because it will `pixi install` under the hood, replacing/superseding `pixi install` from a developer perspective, but I'm happy with either, and maybe it is best to follow the examples.

@jdhughes-usgs @langevin-usgs